src/providers/coinbase.rs

```diff
@@ -115,7 +117,7 @@ impl HistoryProvider for CoinbaseProvider {
                     mined_at: f.created_at,
                     nonce: 1, // TODO: get nonce from somewhere
                     sent_from: "Coinbase".to_string(),
-                    sent_to: address.to_string(),
+                    sent_to: format!("{:#x}", address),
```

Comment:
Lol I think this was already better:

```suggestion
                    sent_to: address.to_string(),
```

Reply:
`address.to_string()` when the address is `H160` returns a short address instead of the full version.